### DISCUSSION TITLE
Is there any way i can contribute to you guys?

### DISCUSSION THREAD

**Discussion by Mos-Def** [termux/termux-app#2592] [Q&A]
I'm new to programming, and I really want to contribute. Just not in code necessarily, do you guys have wiki pages I can help fill out, or maybe even helping you guys with marketing. 

Just in any way I would like to contribute, because of how awesome I find Termux, and it definitely took a lot of work.

**Comment by ghost** [comment]
You can contribute by reporting issues and test stuff hunting bugs for the Termux app. maybe a tester or [give donation to maintainers of the app](https://termux.com/donate) to give devs a motivation

> do you guys have wiki pages I can help fill out, 

The https://wiki.termux.com is restricted to specific people. you may request @Neo-Oli for access if you wish to contribute to the wiki

**Reply by Mos-Def** [reply]
Sorry for the late response! 

Thank you so much for responding, I will definitely be contacting him!

**Reply by Neo-Oli** [reply]
Just create an account an message me the name and I can unlock it.

**Reply by unquote7083** [reply]
@Neo-Oli how to message you?

**Reply by Neo-Oli** [reply]
@slmb3 You can tag me here or write me an  [e-mail](mailto:oli@glow.li).

**Reply by unquote7083** [reply]
@Neo-Oli cool :)
Could you give me write permissions to the wiki? Same name as on here. I wanted to add some missing configs to the Terminal Settings page

**Reply by agnostic-apollo** [reply]
>add some missing configs to the Terminal Settings page

Already done locally. Do something else.

**Reply by Neo-Oli** [reply]
@slmb3 I added you to the trusted user list. You should now be able to change things.

**Comment by SDRausty** [comment]
>give devs a motivation

please
